### ISSUE TITLE
[Question] Is Linux ARM64 supported ?

### ISSUE THREAD

**Issue by julien-faye** [3dem/relion#896]
Hello RELION community.

I’d like to ask whether Linux ARM64 is an officially supported platform? For hardware like AWS Graviton2 / Ampere ?
Do I need to do something special to be able to build and run RELION on ARM64 ?

I searched around but I was not able to find any related information.

Thank you!

**Comment by biochem-fan** [comment]
ARM64 is not officially supported. It should work (albeit slower), but we cannot test or debug. We don't have access to any ARM64 platform.

**Comment by julien-faye** [comment]
Thanks for the quick answer, @biochem-fan !

I have seen OSS projects applying at https://github.com/WorksOnArm/equinix-metal-arm64-cluster for hardware that could be used for testing, debugging, CI purposes!
To apply one just have to open a Github issue.
I hope you consider it!

**Comment by martin-g** [comment]
Hi,

Huawei Cloud is also willing to donate an ARM64 VM for long term usage with no strings attached!

**Comment by dzyla** [comment]
Hi,
I could compile Relion4 on Raspberry Pi, but it is hard to tell if all options work due to limited RAM and CPU. The compilation process works tho.

**Comment by julien-faye** [comment]
Thanks for sharing this info, @dzyla !
I'm glad there are more people interested in ARM64!

**Comment by rbs-sci** [comment]
Around three years ago, I got RELION 3 to compile and everything would work on a 4GB nVidia Jetson Nano (I ran through the cisTEM tutorial dataset as it's small) and the only issue is CTFFIND... which works, provided you manually replace some of the x86 assembly used for speed with platform agnostic C code.

The only caveats were the limited memory (I ended up making a big swap file on a USB3 connected SSD or polishing would crash) and obviously how slow it was (despite CUDA acceleration). On the newer nVidia ARM64 development systems it'd probably do much better... but they are _way_ out of "tinkering" budget.

I'll dig the info out from my archives if anyone is interested.
